app/views/characters/_vote.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="col-md-4" align="right">
+  <%= link_to like_character_path(@character), class:"like-btn", method: :put, remote: :true do %>
+    <i class="fa fa-heart fa-2x"></i>
+  <% end %>
+  <span class="likes-count">Likes: <%= @character.get_upvotes.size %>
+    <span>
+      <% if current_user && (current_user.liked? @character) && (@character.get_upvotes.size > 1)%>
+        , you and others like this character
+      <%elsif current_user && (current_user.liked? @character) && (@character.get_upvotes.size == 1)%>
```

Comment:
Please check your code in more detail and add white space wherever it is needed, it doesn't make sense if I point to every place that needs that change.